test/tbb/test_join_node_key_matching_n_args.cpp

```diff
@@ -0,0 +1,85 @@
+/*
+    Copyright (c) 2005-2023 Intel Corporation
```

Comment:
Not sure whether this file should cover "2005-2023" time range. On one hand, it has been just created, on the other - it contains the code developed earlier than the 2023 year.

Reply:
I would like to keep 2005-2023 since this is a same code.